notification-support/src/main/java/com/expediagroup/streamplatform/streamregistry/core/events/NotificationEventUtils.java

```diff
@@ -51,6 +52,18 @@ public static AvroKey toAvroKeyRecord(Schema schema) {
         .build();
   }
 
+  public static AvroKey toAvroKeyRecord(SchemaKey schemaKey) {
+    return Optional.ofNullable(schemaKey)
+        .filter(sk -> sk.getName() != null)
+        .map(sk -> AvroKey.newBuilder()
+            .setId(sk.getName())
+            .setVersion(null)
```

Comment:
Do we really need to set null? I think I would prefer we don't if null is the default.

Reply:
Good spot. I completely missed this.

The version field shouldn't be present. It's not what I intended with the general purpose key for all types.

```avdl
  record AvroKey {
    string id;
    union {null, int} version = null; // This field shouldn't be present
    AvroKeyType type;
    union {null, AvroKey} parent;
  }
```

I also see below that the parent is not being set. In this case the parent is the domain. 

Examples:

```yaml
schema:
  id: "schema_1"
  type: "SCHEMA"
  parent:
    id: "domain_1"
    type: "DOMAIN"
    parent: null
```

```yaml
stream:
  id: "1"
  type: "STREAM_VERSION"
  parent:
    id: "stream_1"
    type: "STREAM"
    parent:
      id: "domain_1"
      type: "DOMAIN"
      parent: null
```

This is an existing bug that we can probably address separately. Thoughts?

Reply:
Got it, now I got the purpose of `"STREAM_VERSION"`. I'll remove the version field and use the example structure.

Reply:
I ran some samples in my local and these were my results. I think I got the concept :)
```
// Sample key for STREAM
{
  "id": "1",
  "type": "STREAM_VERSION",
  "parent": {
    "com.expediagroup.streamplatform.streamregistry.avro.AvroKey": {
      "id": "demonix_05",
      "type": "STREAM",
      "parent": {
        "com.expediagroup.streamplatform.streamregistry.avro.AvroKey": {
          "id": "dh_test_01",
          "type": "DOMAIN",
          "parent": null
        }
      }
    }
  }
}

// Sample key for SCHEMA
{
  "id": "ccordero_schema",
  "type": "SCHEMA",
  "parent": {
    "com.expediagroup.streamplatform.streamregistry.avro.AvroKey": {
      "id": "my_test_domain",
      "type": "DOMAIN",
      "parent": null
    }
  }
}
```